Source/GTMSessionFetcher.m

```diff
@@ -819,11 +819,12 @@ - (void)beginFetchMayDelay:(BOOL)mayDelay
 #endif  // GTM_DISABLE_FETCHER_TEST_BLOCK
 
 #if GTM_BACKGROUND_TASK_FETCHING
-  // Background tasks seem to interfere with out-of-process uploads and downloads.
-  if (!self.skipBackgroundTask && !self.useBackgroundSession) {
+  // Background tasks seem to interfere with out-of-process uploads and downloads. Background
+  // tasks can only be started if an app exists to support them.
+  id<GTMUIApplicationProtocol> app = [[self class] fetcherUIApplication];
+  if (app && !self.skipBackgroundTask && !self.useBackgroundSession) {
```

Comment:
Since we've got two CPP gates, I'd say this shuffle to ensure the we get an "app" to use is still a good safety net.